charts/rekor/templates/backfillredis/backfill-job.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.backfillredis.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "rekor.backfillredis.fullname" . }}
+{{ include "rekor.namespace" . | indent 2 }}
+  labels:
+    {{- include "rekor.labels" . | nindent 4 }}
+{{- if .Values.backfillredis.annotations }}
+  annotations:
+{{ toYaml .Values.backfillredis.annotations | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.backfillredis.ttlSecondsAfterFinished }}
+  ttlSecondsAfterFinished: {{ .Values.backfillredis.ttlSecondsAfterFinished }}
+{{- end }}
+  template:
+    spec:
+      serviceAccountName: {{ template "rekor.serviceAccountName" . }}
+      restartPolicy: Never
+      automountServiceAccountToken: {{ .Values.backfillredis.serviceAccount.mountToken }}
```

Comment:
`mountToken` is not specified any where.

Reply:
removed it.